container-storage-setup.spec

```diff
@@ -15,10 +15,7 @@ URL:            http://github.com/projectatomic/container-storage-setup/
 Source0: %{git0}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 BuildArch: noarch
 
-BuildRequires:  pkgconfig(systemd)
-
 Requires:       lvm2
-Requires:       systemd-units
```

Comment:
@rhatdan how do we make sure systemctl is installed on the system? container-storage-setup uses that.